cmd/ocitools/validate.go

```diff
@@ -381,6 +386,47 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 	return
 }
 
+func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
+	logrus.Debugf("check linux resources")
+
+	if r.OOMScoreAdj != nil && (int(*r.OOMScoreAdj) < -1000 || int(*r.OOMScoreAdj) > 1000) {
+		msgs = append(msgs, fmt.Sprintf("OOMScoreAdj range is from -1000 to 1000"))
```

Comment:
On Wed, Aug 17, 2016 at 10:48:39PM -0700, Ma Shimiao wrote:
> > @@ -381,6 +386,47 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
> >  	return
> >  }
> >  
> > +func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
> > +	logrus.Debugf("check linux resources")
> > +
> > +	if r.OOMScoreAdj != nil && (int(*r.OOMScoreAdj) < -1000 || int(*r.OOMScoreAdj) > 1000) {
> > +		msgs = append(msgs, fmt.Sprintf("OOMScoreAdj range is from -1000 to 1000"))
> 
> If so, I'm glade to remove these short of value checks

Filed against v1.0.0.rc1 as #197.  Figuring out the right schema URL
for a -dev version is unclear to me, so we might have to wait on
forward-porting until runtime-spec cuts a v1.0.0-rc2.